config/environments/test.rb

```diff
@@ -37,6 +37,11 @@
   # Store uploaded files on the local file system in a temporary directory.
   config.active_storage.service = :test
 
+  Rails.application.routes.default_url_options[:host] = "localhost:3000"
+  config.action_mailer.default_url_options = { host: "localhost:3000" }
+
+  config.active_job.queue_adapter = :inline
+
```

Comment:
I believe the canonical way to use RSpec to test notification queues is to use `have_enqueued_job.on_queue`... but for the life of me I could not get it to report anything but an empty queue. The only way I saw jobs show up in queues was by setting `config.active_job.queue_adapter` to `inline` instead of `test`. But when I did that the `have_enqueued` matchers complained. :man_shrugging: